src/bindings/Vscode.mli

```diff
@@ -101,7 +101,7 @@ end
 module WorkspaceConfiguration : sig
   type t
 
-  val get : t -> string -> Js.Json.t option
+  val get : t -> string -> Js.Json.t Js.Nullable.t
```

Comment:
How about we keep the signature but change the implementation to do the unwrapping?

I'd prefer not to leak bucklescript implementation details whenever it's possible